core-java/src/main/java/org/spine3/server/EntityRepository.java

```diff
@@ -41,41 +45,53 @@
 
     @Nullable
     @Override
-    protected EntityStorage<I, M> getStorage() {
```

Comment:
As discussed vocally, it's not going to cause problems because we control the passed type from an EntityRepository.